docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+|                                                                                                                                                                                                     | **Bitwarden design best practice:** Icons and text that is shortened should always have a `title` attribute to provide additional context.                                                                                                                                                                                                                                                                                                                                                         |
+
+## Keyboard Navigation
+
+| WCAG Success Criteria                                                                                                                                                       | Description                                                                                                                                                              |
+| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)                                                                                                 | All mouse interactive elements on a page should be accessible via keyboard navigation.                                                                                   |
+| [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)<br/><br/>[2.1.2 No keyboard trap](https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap) | If a popup (especially in the case of multiple dialogs) is opened, be sure to test that focus is moving to and from each dialog correctly and there is no keyboard trap. |
+| [2.4.7 Focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)                                                                                       | Each interactive element should have a visible focus indicator to assist user navigating the page by keyboard.                                                           |
+
+## Screen Reader
+
+Users with low vision or blindness will often navigate products using keyboard navigation and a
+screen reader. This results in an audio read out of the page’s content, often with additional
+instruction depending on what semantic element is in focus.
+
+To understand more about screen reader and keyboard navigation, turn on `voice over` or download
+`NVDA` and try navigating your device with keyboard and screen reader.
```

Comment:
Is it worth linking to https://www.nvaccess.org/download/?

Reply:
Yeah, I can do that and expand on voice over as well